package.json

```diff
@@ -83,6 +86,9 @@
     "warning": "^3.0.0"
   },
   "peerDependencies": {
+    "@gandi/babel-preset-gettext": "*",
+    "babel-cli": "*",
```

Comment:
I just re-read why peerdeps are not automatically installed and well we have an issue here:

Facts:
* In case of the create-react-app or gulp generator or similar: `babel-cli` is not installed in the root node_modules
* `extract_message` refers to babel-cli in root node_modules
* npm2 install deps in relative node_modules or automatically install peerdeps
* npm3 doesn't install peerdeps but print warnings

Issue:
* npm2 & 3 fails to launch `extract_message` if peerdeps is not declared
* npm3 fails if `babel-cli` is not in deps
* install fails if user set a version of babel-cli we don't support
* we can't use relative path to determine where the babel-cli binary is stored
* this issue is the same for po2json

The position of npm author on this point is, it's up to the developer to install the version he wants for peerDeps